proto/evmos/epochs/genesis.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package evmos.epochs.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/tharsis/evmos/x/epochs/types";
+
+message EpochInfo {
+  string identifier = 1;
+  google.protobuf.Timestamp start_time = 2 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"start_time\""
+  ];
+  google.protobuf.Duration duration = 3 [
```

Comment:
@ramacarlucho The is already an `epoch_duration` in the Params which can be changed by governance. Can we delete this one?